### DISCUSSION TITLE
Page Arrangement

### DISCUSSION THREAD

**Discussion by Utkub24** [Duck-Tech-Dev/api-winnerduck#6] [Ideas]
From @ErtyumPX 

Considering that we will use only 1 domain, which we will call `BASE` for now.

Suggested endpoints:

**For guests**
- Enter raffle: `BASE/`
- Raffle participating page: `BASE/raffle/[id]`

**For users**
- General panel: `BASE/panel`
- Raffle page to be projected during the event: `BASE/panel/raffle/[id]`

**Comment by erthium** [comment]
Now that I think about it, after talking with @HuseyinSimsek7904, this part IS completely about front-end and does not make sense to be in this repo :) let's ot forget to transfer it after starting the front-end repo...

**Reply by HuseyinSimsek7904** [reply]
> IS

Is the capitalization intentional?

**Reply by erthium** [reply]
yes, it actually is intentional :))

**Comment by erthium** [comment]
Discussion is moved to the fronted part, to `app-winnderduck` repository
